code/game/objects/objs.dm

```diff
@@ -245,7 +245,7 @@
 		return
 	..()
 
-/obj/examine(mob/user)
+/obj/examine(mob/user, var/distance = -1, var/infix = "", var/suffix = "")
```

Comment:
What is the purpose of this change?

Reply:
Same as above, I had to expand the examine procs so that the show_blood = FALSE would line up correctly on the end.  Though I suppose that given that the child procs below obj/examine use their arguments correctly, removing the extra arguments from obj/examine could work.  I can't remember if that made it not work right in testing or not.